test/godeps.list

```diff
@@ -1,8 +1,5 @@
-github.com/go-macaroon-bakery/macaroon-bakery/v3/bakery
 github.com/go-macaroon-bakery/macaroon-bakery/v3/httpbakery
```

Comment:
What's still using /httpbakery? I'd have expected it to also go away with this change.

Reply:
I was using the `OpenWebBrowser` function from httpbakery, but added a similar function to `util.go` now.